sn_transfers/src/client_transfers/mod.rs

```diff
@@ -87,5 +84,5 @@ pub struct CreatedDbc {
     pub dbc: Dbc,
     /// This is useful for the sender to know how much they sent to each recipient.
     /// They can't know this from the dbc itself, as the amount is encrypted.
-    pub amount: RevealedAmount,
+    pub amount: Token,
```

Comment:
I think we can remove this field now that the `Token` can be obtained directly from the `Dbc`. Maybe this entire struct can be removed as a result?

Reply:
sounds good, I'll give it a try